Intellij IDEA Tutorial.md

```diff
@@ -0,0 +1,95 @@
+# IntelliJ IDEA Tutorial
+
+## **1. Download IntelliJ IDEA Community Edition**
+
+IntelliJ IDEA Community Edition is an open-source version of IntelliJ IDEA, a premier IDE for Java, Scala and other JVM-based programming languages. [You can download it from the official website](https://www.jetbrains.com/idea/download).
+
+## **2. Install the Scala plugin**
+After the installation, launch Intellij. You should see the welcome screen. Click on **plugins** and then search for "scala" in the list as illustrated bellow. Install the latest Scala (from JetBrains) plugin for Intellij.
+**Note**: You can do the same from the opened IDE with **File** > **Settings** > **Plugins**.  
+
+![](./intellij-tutorial-images/scala-plugin.png)
+
+Note that Scala plugin requires restart to complete installation. Thus, **restart Intellij** before following the next steps.
+
+## **3. Create a project**
+From the welcome screen: **Projects** > **New Project**. As illustrated bellow select **Scala** > **sbt**.
+
+![](./intellij-tutorial-images/create-project-1.png)
+
+Now, rename the project and make sure that the selected JDK version is either 1.8 or 11. Moreover, check that the Scala version is 2.13.x and that the sbt version used is higher than 1.x.x. If you don't see the targeted version, please make sure that you have followed the [tools setup](https://www.coursera.org/learn/progfun1/supplement/BNOBK/tools-setup-please-read). After that click **finish** and wait for SBT being done processing.
+
+
+![](./intellij-tutorial-images/create-project-2.png)
+
+**Note**: You can always change the JDK version in Intellij by going to **File** > **Project Structure** > **Project**. And then you can select the desired JDK under **Project SDK**.
+ 
+**Note**: The first time we create or import a SBT project, Intellij may not show the structure of the project right away. This happens because Intellij decides to delay this task until it has successfully executed sbt and downloaded all the required dependencies. Once that's done, the usual `src` folder and their subfolders will be created.
```

Comment:
```suggestion
**Note**: The first time we create or import a SBT project, IntelliJ may not show the structure of the project right away. This happens because IntelliJ decides to delay this task until it has successfully executed sbt and downloaded all the required dependencies. Once that's done, the usual `src` folder and their subfolders will be created.
```